### DISCUSSION TITLE
Add feature for Teams

### DISCUSSION THREAD

**Discussion by jadonbrownza** [vitodeploy/vito#322] [Ideas]
Perfect for agencies, that offer server management to multiple customers, 1 log-in.

Teams have access only to their servers.

**Comment by saeedvaziry** [comment]
We have Projects

**Comment by jadonbrownza** [comment]
Some Teams can have multiple Projects

**Comment by saeedvaziry** [comment]
Vito is not built for that purpose. it is not a SaaS. its a self hosted app.

I believe it will bring more complexity to the system.

**Comment by jadonbrownza** [comment]
In the context of Projects then, is it already possible to assign certain users to certain projects?

**Reply by saeedvaziry** [reply]
Yes
